packages/components/src/alignment-matrix-control/README.md

```diff
@@ -54,21 +54,22 @@ Accessible label. If provided, sets the `aria-label` attribute of the underlying
 
 If provided, sets the default alignment value.
 
-- Type: `string`
+- Type: `AlignmentMatrixControlValue`
```

Comment:
I wasn't sure (and I found inconsistent results when comparing to other components) how best to list this in the README. `string` felt too broad, listing the accepted strings felt messy, and `AlignmentMatrixControlValue` feels like it lacks context to someone reading the doc for the first time.

I went with the specific type name, `AlignmentMatrixControlValue` because it felt the most accurate, but I'd love some input on the preferred approach. If needed, I'll update this and the other references to this type later in the doc.

Reply:
Exactly, this is just one of those things we have to weigh on a case-by-case basis. I think your choice is reasonable. Good thing the enums are showing up nicely in the Storybook though!